addons/cilium/cilium.yaml

```diff
@@ -82,7 +83,11 @@ data:
 
   # Enable IPv6 addressing. If enabled, all endpoints are allocated an IPv6
   # address.
```

Comment:
The check would fail if somebody wanted two different IPv4 ranges for their pods? 
We don't allow setting more than one range from the UI so not sure if this will ever be an issue.

Reply:
that is not allowed in k8s - and should not be allowed by KKP validation webhook

Reply:
Cool. Good to know.